src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2065,6 +2061,12 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
         for (FeedbackQuestionAttributes qn : allQuestions) {
             allQuestionsMap.put(qn.getId(), qn);
         }
+        
+        boolean isInSection = Boolean.parseBoolean(params.get("inSection"));
+        boolean isToSection = Boolean.parseBoolean(params.get("toSection"));
+        boolean isFromSection = Boolean.parseBoolean(params.get("fromSection"));
+        boolean isComplete = params.get("range") != null ? false : true;
```

Comment:
There's a PMD rule that will catch this, but let's fix it now. `params.get("range") == null` will do the trick.